api/equipment/schema.py

```diff
@@ -3,12 +3,42 @@
 from graphene import relay, Schema
 from graphene_sqlalchemy import (SQLAlchemyObjectType, 
                                  SQLAlchemyConnectionField)
-
 from api.equipment.models import Equipment as EquipmentModel
+from utilities.utility import validate_empty_fields
 
 
 class Equipment(SQLAlchemyObjectType):
     
     class Meta:
         model = EquipmentModel
-        interfaces = (relay.Node, )
\ No newline at end of file
+        interfaces = (relay.Node, )
+
+class UpdateRoomResource(graphene.Mutation):
+    class Arguments:
+        name = graphene.String()
+        room_id = graphene.Int()
+        resource_id = graphene.Int()
+
+    equipment = graphene.Field(Equipment)
```

Comment:
@owenbob, it looks like you still maintained the use of "equipment". You may need to change it to "resource".

Reply:
Thanks for noting this. I have updated it to resources